getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "id": "ebd83bd8-da43-4c2b-a07f-a1bd0ecd0d19",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The status code is: 200\n"
+     ]
+    }
+   ],
+   "source": [
+    "import base64\n",
+    "import httpx\n",
+    "\n",
+    "url = \"https://upload.wikimedia.org/wikipedia/commons/b/bf/Aoraki_Mount_Cook.JPG\"\n",
+    "\n",
+    "# Use an HTTP get request to retrieve the image\n",
+    "response = httpx.get(url)\n",
+    "\n",
+    "# Check the request was successful\n",
+    "if response.status_code == 200:\n",
+    "    # Extract the binary image data\n",
+    "    binary_image_data = response.content\n",
+    "\n",
+    "    # Encode the binary_image_data into base 64 so that it's in bytes\n",
+    "    # This allows it to be transmitted in JSON or text format\n",
+    "    byte_image_data = base64.b64encode(binary_image_data)\n",
+    "\n",
+    "    # Decode the data into UTF-8 to allow for to make the data workable\n",
+    "    # Web applications and models require it in a workable format\n",
+    "    # UTF-8 is a fairly standardised format\n",
+    "    image_data = byte_image_data.decode(\"utf-8\")\n",
+    "\n",
+    "print(f'The status code is: {response.status_code}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65f20898-73d6-4fed-9433-bb086019f1a7",
+   "metadata": {},
+   "source": [
+    "When using data of different types, such as text in the `SystemMessage` and a file in the `HumanMessage`, it's necessary to specify a type header so the model knows how to interpret the data.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "id": "fdb153be-bcb0-4a61-902a-3394cfb1206d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'This image depicts a stunning mountainous landscape with a snow-capped peak reflected in a calm, clear lake. The mountains are rugged and covered in snow, suggesting a cold and possibly remote location. The clear blue sky and the reflection in the water add to the serene and picturesque quality of the scene. The foreground features some rocks along the edge of the lake, enhancing the natural beauty of the setting. This type of scenery is often found in high-altitude regions, such as the Himalayas, the Alps, or the Southern Alps in New Zealand.'"
+      ]
+     },
+     "execution_count": 60,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "messages = [  \n",
+    "    SystemMessage(content=[{\"type\": \"text\", \"text\": \"describe the image location\"}]),\n",
+    "    HumanMessage(\n",
+    "    content=[\n",
+    "        {\n",
+    "            \"type\": \"image_url\",\n",
+    "            \"image_url\": {\"url\": f\"data:image/jpeg;base64,{image_data}\"},\n",
```

Comment:
I think it may also be possible just to pass the URL here, should we mention that as well?

Reply:
I didn't realise that! I've updated the section to include a part on passing the url directly. Let me know if you feel the section is too long now as I can just delete the image encoding part since it's not as relevant.

Reply:
I think it might be good to keep it short and just mention passing the URL directly